docs/advanced-topics/docker-alternatives.rst

```diff
@@ -122,7 +122,7 @@ Podman
 Podman is a daemon-less alternative of Docker that allows end users to run Docker/Open Container Initiative (OCI) containers without root privileges.
 
 Podman can be installed following the instructions
-`here <https://podman.io/getting-started/installation>`__.
```

Comment:
Hmmm, the old link works and the new one doesn't. Possible copy-paste error in this PR or a transient error on their site?